src/config.ts

```diff
@@ -59,6 +61,11 @@ export default {
   redis: {
     prefix: getStringOption(process.env.REDIS_PREFIX, 'LEARNINGLOCKER'),
     url: getStringOption(process.env.REDIS_URL, 'redis://127.0.0.1:6379/0'),
+    tlsIsEnabled: getBooleanOption(process.env.REDIS_TLS_IS_ENABLED, false),
+    db: getNumberOption(process.env.REDIS_DB, DEFAULT_REDIS_DB),
+    password: getStringOption(process.env.REDIS_PASSWORD, ''),
+    host: getStringOption(process.env.REDIS_HOST, '127.0.0.1'),
```

Comment:
@PrinceWaune could the default value for REDIS_HOST be a constant variable like we have for `DEFAULT_REDIS_PORT`?

Reply:
I guess the `REDIS_URL` could be the same and `REDIS_PREFIX` so feel free to consider this as a `Nit` suggestion! :]

Reply:
Also, can you please add a description to this PR please?

Reply:
Ah, gotcha!